### DISCUSSION TITLE
Connect Exandra repo with AWS Keyspaces

### DISCUSSION THREAD

**Discussion by kennethtegrado** [vinniefranco/exandra#74] [Q&A]
Have you tried connecting Exandra with AWS Keyspaces? I am having a hard time finding any resources to do this. As much as possible, we want aws managed services for our Cassandra database. Thanks

**Comment by vinniefranco** [comment]
It looks like you can connect to key spaces using csql clients, so assuming
permissions are correct then you would connect using the AWS provided
connection string

**Reply by kennethtegrado** [reply]
Does this mean that I can't use Xandra and Exandra for my use case? Like do I have to implement the connection then pool that connection to my whole phoenix application

**Comment by vinniefranco** [comment]
Oh, no. It means it should connect like any other instance of Cassandra
